daemon/config/config.go

```diff
@@ -317,15 +317,15 @@ func GetConflictFreeLabels(labels []string) ([]string, error) {
 		if len(stringSlice) > 1 {
 			// If there is a conflict we will return an error
 			if v, ok := labelMap[stringSlice[0]]; ok && v != stringSlice[1] {
-				return nil, fmt.Errorf("conflict labels for %s=%s and %s=%s", stringSlice[0], stringSlice[1], stringSlice[0], v)
+				return nil, errors.Errorf("conflict labels for %s=%s and %s=%s", stringSlice[0], stringSlice[1], stringSlice[0], v)
```

Comment:
Any particular reason for changing out the stdlib `fmt.Errorf` with a third-party `errors.Errorf`?

Reply:
This package was already using `pkg/errors` for all other errors, so aligning with other errors produced. 

`pkg/errors` also gives us a stack-trace (something Go's own error package unfortunately doesn't give); I realise it was archived (but mostly "feature complete"), but we were considering to reach out to the author to see if they're open to handing over maintainership.